woocommerce/woocommerce
# Checkout: Add updated shipping method notice.

### Changes proposed in this Pull Request:

Closes #50558 

Adds a styled grey message when the shipping address has not been filled. Updates wording in this case and also in the case where there is not a shipping option available for the address. See #50558 for exact details.



### How to test the changes in this Pull Request:

1. Go through to checkout as a shopper, 
2. While the address is not filled out, see that there is a new message under `Shipping Options` with the correct wording `Enter a shipping address to view shipping options`

<img width="928" alt="Screenshot 2024-11-06 at 12 42 41 PM" src="https://github.com/user-attachments/assets/cf9a46a5-db71-4088-a1f0-78fd2625345b">


3. Now fill out the shipping address with an address that does not match your shipping zones you have setup
4. Now you should see the warning notice with updated wording: `No shipping options are available for this address. Please verify the address is correct or try a different address.`

<img width="847" alt="Screenshot 2024-11-06 at 12 43 11 PM" src="https://github.com/user-attachments/assets/76957327-7247-4ea4-9056-e1c9a4a5a5f8">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Checkout: Adjust styling and wording of "No delivery address" and "No shipping option" messages.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
